.circleci/config.yml

```diff
@@ -19,7 +19,7 @@ executors:
   ios-executor:
     working_directory: ~/ios
     macos:
-      xcode: 12.0
+      xcode: 12.5.1
```

Comment:
we were getting `definition conflicts with previous value` errors pointing towards the eTags stuff in `purchases-ios` for some reason. 
it looks like it was either a bug in Xcode 12.0, or in one of the other dependencies associated with the CircleCI instance for `xcode 12.0`. In any case, it gets fixed by bumping the version number here, and it works correctly locally.